app/main.py

```diff
@@ -2,5 +2,17 @@
 
 
 def cache(func: Callable) -> Callable:
-    # Write your code here
-    pass
+    funcs = {}
+
+    def wrapper(*args, **kwargs) -> Callable:
```

Comment:
You could use a decorator from `functools` to preserve the wrapped function name and docstring.